### PR TITLE
styled graphs

### PR DESCRIPTION
**Pull request by theswerd**
### Update stats graph styling
Styles the stats graph to match the Figma.

#### Screenshots
!
Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-13 at 12 05 12
Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-13 at 12 05 14
Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-13 at 12 05 16
Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-13 at 12 05 19

#### How did you test the change?
* [x] iOS Simulator
* [ ] Android Emulator
* [ ] iOS Device
* [ ] Android Device
* [ ] `curl` to a dev App Engine server

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [ ] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

File: client/flutter/lib/components/recent_numbers_graph.dart [hunk 1]
```diff
@@ -70,9 +60,113 @@ class _RecentNumbersGraphState extends State<RecentNumbersGraph> {
...
+        return "cases";
+      case DataDimension.deaths:
+        return "deaths";
+      default:
+        return "";
+    }
+  }
+}
+
+double verticalInterval({@required List timeseries, @required String key}) {
```
Comment (hspinks) [inline client/flutter/lib/components/recent_numbers_graph.dart:163]: is this still used?

Comment (theswerd) [reply client/flutter/lib/components/recent_numbers_graph.dart:163]: it is not

**Comment by advayDev1** [comment]
```
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
I/flutter (24295): The following NoSuchMethodError was thrown building RecentNumbersGraph(dirty, state:
I/flutter (24295): _RecentNumbersGraphState#242dc):
I/flutter (24295): The getter 'last' was called on null.
I/flutter (24295): Receiver: null
I/flutter (24295): Tried calling: last
I/flutter (24295): 
I/flutter (24295): The relevant error-causing widget was:
I/flutter (24295):   RecentNumbersGraph
I/flutter (24295):   file:///Users/madvay/src/who-app/who-covid-app/client/flutter/lib/pages/main_pages/recent_numbers.dart:120:28
I/flutter (24295): 
I/flutter (24295): When the exception was thrown, this was the stack:
I/flutter (24295): #0      Object.noSuchMethod (dart:core-patch/object_patch.dart:53:5)
I/flutter (24295): #1      _RecentNumbersGraphState.build (package:who_app/components/recent_numbers_graph.dart:115:49)
I/flutter (24295): #2      StatefulElement.build (package:flutter/src/widgets/framework.dart:4619:28)
I/flutter (24295): #3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4502:15)
I/flutter (24295): #4      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4675:11)
```

**Comment by theswerd** [comment]
@advayDev1 fixed

**Pull request merged**